shell/browser/api/electron_api_web_frame_main.h

```diff
@@ -153,8 +153,6 @@ class WebFrameMain final : public gin::Wrappable<WebFrameMain>,
   content::FrameTreeNodeId frame_tree_node_id_;
   content::GlobalRenderFrameHostToken frame_token_;
 
-  raw_ptr<content::RenderFrameHost> render_frame_ = nullptr;
```

Comment:
Looks like this is the only `raw_ptr` in this header. Can we remove `#include "base/memory/raw_ptr.h"` on line 12?